_join/computerscience.md

```diff
@@ -2,9 +2,11 @@
 layout: page 
 title: Computer Science
 permalink: /join/cs/
+cta_background: /assets/images/cs_background.png
 ---
 
 <!-- Pic -->
+![](/assets/images/2019_sar_cs.png)
```

Comment:
Can we add some alt text for our visually impaired visitors?

Reply:
After reviewing the Kramdown docs, yes we can! I'll get on that tomorrow morning.